saleor/graphql/order/mutations/order_note_update.py

```diff
@@ -0,0 +1,58 @@
+import graphene
+
+from ....core.tracing import traced_atomic_transaction
+from ....order import OrderEvents, events, models
+from ....permission.enums import OrderPermissions
+from ...app.dataloaders import get_app_promise
+from ...core import ResolveInfo
+from ...core.descriptions import ADDED_IN_314, PREVIEW_FEATURE
+from ...core.doc_category import DOC_CATEGORY_ORDERS
+from ...core.types import OrderError
+from ...plugins.dataloaders import get_plugin_manager_promise
+from ..types import Order, OrderEvent
+from .order_note_common import OrderNoteCommon
+from .utils import get_webhook_handler_by_order_status
+
+
+class OrderNoteUpdate(OrderNoteCommon):
+    order = graphene.Field(Order, description="Order with the note updated.")
+    event = graphene.Field(OrderEvent, description="Order note updated.")
+
+    class Arguments(OrderNoteCommon.Arguments):
+        id = graphene.ID(
+            required=True,
+            description="ID of the note.",
+            name="note",
+        )
+
+    class Meta:
+        description = "Updates note of an order." + ADDED_IN_314 + PREVIEW_FEATURE
+        doc_category = DOC_CATEGORY_ORDERS
+        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        error_type_class = OrderError
+
+    @classmethod
+    def perform_mutation(  # type: ignore[override]
+        cls, _root, info: ResolveInfo, /, *, id: str, input
+    ):
+        qs = models.OrderEvent.objects.filter(
+            type__in=[OrderEvents.NOTE_ADDED, OrderEvents.NOTE_UPDATED]
+        ).select_related("order")
+        order_event_to_update = cls.get_node_or_error(
+            info, id, only_type=OrderEvent, qs=qs
+        )
+        order = order_event_to_update.order
+        cleaned_input = cls.clean_input(info, order, input)
+        app = get_app_promise(info.context).get()
+        manager = get_plugin_manager_promise(info.context).get()
+        with traced_atomic_transaction():
```

Comment:
I think we could use a normal transaction here instead of a traced transaction.

Reply:
Could you explain why? I'm using the `traced_atomic_transaction` just as it was in the original `OrderAddNote` mutation.